apps/homepage2/package.json

```diff
@@ -52,13 +53,11 @@
     "uuid": "^9.0.0"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.24.5",
```

Comment:
I am not sure that I removed all unnecessary dependencies. Where did you use this dependency?

Reply:
can't remember +1 to remove